src/components/profileNavbar.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Navbar, Nav, Button } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import '../styles/App.css';
+
```

Comment:
Rebase correctly...no profile navbar in develop.

Reply:
@IssaIan  already fixed